
====== START ORIO ======

----- begin reading the source file: haxpy3.ancc.c -----
----- finished reading the source file -----

----- begin parsing annotations -----
----- finished parsing annotations -----

----- begin optimizations -----
orio.main.tspec.eval: 16: invalid reference: "RANGE"
  File "/home/pbalapra/Libs/bin/orcc", line 24, in <module>
    orio.main.orio_main.start(sys.argv, orio.main.orio_main.C_CPP)

  File "/fusion/gpfs/home/pbalapra/Libs/lib/python2.6/site-packages/orio/main/orio_main.py", line 97, in start
    optimized_code_seq = odriver.optimizeCodeFrags(cfrags,{}, False)

  File "/fusion/gpfs/home/pbalapra/Libs/lib/python2.6/site-packages/orio/main/opt_driver.py", line 168, in optimizeCodeFrags
    cur_seq = self.__optimizeCodeFrag(cf, perf_params)

  File "/fusion/gpfs/home/pbalapra/Libs/lib/python2.6/site-packages/orio/main/opt_driver.py", line 91, in __optimizeCodeFrag
    cfrag.cfrags)

  File "/fusion/gpfs/home/pbalapra/Libs/lib/python2.6/site-packages/orio/main/tuner/tuner.py", line 45, in tune
    tinfo = self.__extractTuningInfo(module_body_code, line_no)

  File "/fusion/gpfs/home/pbalapra/Libs/lib/python2.6/site-packages/orio/main/tuner/tuner.py", line 205, in __extractTuningInfo
    tinfo = orio.main.tspec.tspec.TSpec().parseSpec(code, line_no)

  File "/fusion/gpfs/home/pbalapra/Libs/lib/python2.6/site-packages/orio/main/tspec/tspec.py", line 24, in parseSpec
    stmt_seq = eval.TSpecEvaluator().evaluate(stmt_seq)

  File "/fusion/gpfs/home/pbalapra/Libs/lib/python2.6/site-packages/orio/main/tspec/eval.py", line 292, in evaluate
    return self.__evaluate(stmt_seq, {}, {})

  File "/fusion/gpfs/home/pbalapra/Libs/lib/python2.6/site-packages/orio/main/tspec/eval.py", line 279, in __evaluate
    e = [self.__evaluate(s, env, name_space) for s in stmt]

  File "/fusion/gpfs/home/pbalapra/Libs/lib/python2.6/site-packages/orio/main/tspec/eval.py", line 259, in __evaluate
    e = self.__evalDef(stmt, env, name_space)

  File "/fusion/gpfs/home/pbalapra/Libs/lib/python2.6/site-packages/orio/main/tspec/eval.py", line 143, in __evalDef
    stmt_seq = self.__evaluate(stmt_seq, env, name_space)

  File "/fusion/gpfs/home/pbalapra/Libs/lib/python2.6/site-packages/orio/main/tspec/eval.py", line 279, in __evaluate
    e = [self.__evaluate(s, env, name_space) for s in stmt]

  File "/fusion/gpfs/home/pbalapra/Libs/lib/python2.6/site-packages/orio/main/tspec/eval.py", line 263, in __evaluate
    e = self.__evalParam(stmt, env, name_space)

  File "/fusion/gpfs/home/pbalapra/Libs/lib/python2.6/site-packages/orio/main/tspec/eval.py", line 188, in __evalParam
    err('orio.main.tspec.eval: %s: invalid reference: "%s"' % (rhs_line_no, vname))

  File "/fusion/gpfs/home/pbalapra/Libs/lib/python2.6/site-packages/orio/main/util/globals.py", line 169, in err
    Globals().logger.error(errmsg + '\n' + '\n'.join(traceback.format_stack()))

